dkg/pedersen/pedersen.go

```diff
@@ -488,6 +491,230 @@ func (w worker) work(jobIndex int) error {
 	return nil
 }
 
+// EncryptSecret implements dkg.Actor.
+func (a *Actor) EncryptSecret(msg []byte) (U kyber.Point, Cs []kyber.Point) {
```

Comment:
@si-co this is the function I was talking to you about.
Do you see how we could merge `EncryptSecret` and `VerifiableEncrypt` ?